modules/perception/production/conf/perception/camera/obstacle.pt

```diff
@@ -42,15 +42,15 @@ postprocessor_param {
 lane_param {
   lane_detector_param {
     plugin_param {
-      name : "DenselineLaneDetector"
+      name : "DarkSCNNLaneDetector"
       root_dir : "/apollo/modules/perception/production/data/perception/camera/models/lane_detector/"
-      config_file : "config.pt"
+      config_file : "config_darkSCNN.pt"
     }
     camera_name : "front_6mm"
   }
   lane_postprocessor_param {
-      name : "DenselineLanePostprocessor"
-      root_dir : "/apollo/modules/perception/production/data/perception/camera/models/lane_postprocessor/denseline/"
+      name : "DarkSCNNLanePostprocessor"
```

Comment:
Make every effort to keep careful and consistent coding style across the whole repo, especially in one file. Double check after "copy-and-paste", such as the indents.